### ISSUE TITLE
Remove only unused methods?

### ISSUE THREAD

**Issue by salmanm** [lodash/lodash-webpack-plugin#114]
I tried this plugin and its working great. However I was wondering if there's a way this plugin can be configured to remove *only unused* methods!? I find it really inconvenient to manually set which feature-set I want! Also, developers might start using a feature-set which we opted-out of and end-up spending hours figuring out what's wrong until they realise that its getting removed by this plugin.

I haven't dig deep yet, but just wanted to ask if its possible to detect only used functions and remove everything else! This might also save some extra space in cases where I use say only `get` but not `set`, so set should be removed until I use it.

Did I make any sense?

**Comment by jdalton** [comment]
Hi @salmanm!

Since the plugin opts out of all heavy functionality by default trying to guess which features folks want to toggle isn't doable. Basic functionality of all methods should exist even with features off. If you run into a case where even basic functionality doesn't work then it's likely a bug.

**Comment by salmanm** [comment]
There it is https://github.com/gajus/react-css-modules/issues/131#issuecomment-270867981

Still my original question is that can it remove absolutely only functions that aren't used? Without removing whole set of functions. As mentioned in the above conversation, we have to keep  `collections` and `shorthands`.

Also, I tried with all features off i.e. with `new LodashModuleReplacementPlugin()` and I found that a call to `_.get` returns `undefined` to me.
